intera_tools_description/urdf/electric_gripper/fingers/basic_hard_tip.xacro

```diff
@@ -20,8 +20,8 @@
       </collision>
       <inertial>
         <origin rpy="0 0 ${pi*(g_reflect+1)/2}" xyz="0 0 0"/>
-        <mass value="2e-06"/>
-        <inertia ixx="1.1e-09" ixy="0" ixz="0" iyy="1.1e-09" iyz="0" izz="1.1e-09"/>
+        <mass value="0.01"/>
```

Comment:
Is it possible for this to be '0.001' kg instead (1 gram instead of 10 grams)?

I recognize that 10 grams is already barely anything, but when you add them together, it starts getting close to possibly making numbers seem just a bit off. 1 gram would be nice and well outside any range of shown numbers.